### PR TITLE
Feature/list view of surveys

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**
Implement a list view of all surveys and a detail view

### CODE REVIEW

**Review by JannikUekoetter — COMMENTED** [review]
Looks great! I have noted one small thing.

File: components/EvaluationCard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+export default function EvaluationCard({ question, yes, no, na }) {
+    const total = yes + no + na;
+  return (
+    <StyledEvaluationCard>
+      <h2>{question}</h2>
```
[the remark this answers is not in the document]
Comment (Ozanyurdakul98) [reply components/EvaluationCard.js:7]: that is true

**Review by Ozanyurdakul98 — APPROVED** [review]
Only thing that should be changed, is described by Jannik

File: components/EvaluationCard.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Roland-Hufnagel) [reply components/EvaluationCard.js:7]: you are right!

[the remark this answers is not in the document]
Comment (Roland-Hufnagel) [reply components/EvaluationCard.js:7]: you are doch nicht right. Sorry. There is already a h1 that comes from the [id].js. There is a list and this component is rendered there multiple times. So the h2 in this situation is the right one.

**Pull request merged**